### PR TITLE
[Auctions] Use is_closed field in artwork grid

### PR DESCRIPTION
**Pull request by mennenia**
https://artsyproduct.atlassian.net/browse/IOS-90

The `is_open` field is like an internal Artsy auction field, essentially saying that the work is open to bid on Artsy. This works for live auctions, as you can already put bids in before it's started.

So what happened in the issue is that timed auctions can be published before the auction opens, the works aren't "biddible" yet, yet the auction also hasn't closed either. 

So the fix here is using the `is_closed` field, which is only true when the auction has properly closed, and as such we can then and only then display "Auction Closed". 

What the data looks like in metaphysics for the type of work that caused the bug:

screen shot 2018-02-22 at 13 41 35

### CODE REVIEW

File: src/lib/Components/ArtworkGrids/__tests__/__snapshots__/Artwork-tests.tsx.snap [hunk 1]
```diff
@@ -1,5 +1,176 @@
...
+            "color": "#666666",
+            "fontSize": 12,
+          },
+          Object {
+            "fontFamily": "AGaramondPro-Regular",
+          },
+        ]
+      }
+    >
+      $200 (1 bid)
```
Comment (mennenia) [inline src/lib/Components/ArtworkGrids/__tests__/__snapshots__/Artwork-tests.tsx.snap]: monies!

**Comment by sweir27** [comment]
This logic looks okay to me! Thanks for the fix 😄

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by mennenia** [comment]
@sweir27 Thank you for having a look and getting a staging auction up! 🙏

**Comment by mennenia** [comment]
@orta could you have a look over as well and merge if you think it's ok? :)

**Comment by orta** [comment]
Yep - this makes sense

**Pull request merged**